src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-secondaryDropDown/Components/secondaryDropDown.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import * as S from '../styles';
+import * as H from '../../typography';
+import { Details } from '../constants';
+
+
+export default function SecondaryDropDown() {
```

Comment:
To be consistent to code structure
```suggestion
export const SecondaryDropDown: React.FC = () => {
```

Change accordingly in ```Components/index.ts```

Also make an entry in ```ignitus-Shared/index.ts```,  for easy access.

Reply:
Sure!